packages/aragon-messenger/src/providers/WindowMessage.js

```diff
@@ -17,12 +17,12 @@ export default class PostMessage extends Provider {
   /**
    * An observable of messages being sent to this provider.
    *
-   * @memberof PostMessage
+   * @memberof WindowMessage
    * @instance
    * @returns {Observable}
    */
   messages () {
-    return Observable.fromEvent(window, 'message')
+    return Observable.fromEvent(window, 'message', false)
```

Comment:
The `false` bit passes `false` into `addEventListener()`'s last argument. It's not strictly necessary for newer browsers, but might as well include it.